fdbserver/DataDistribution.actor.cpp

```diff
@@ -2789,7 +2802,9 @@ ACTOR Future<Void> zeroServerLeftLogger_impl(DDTeamCollection* self, Reference<T
 		bytesLost += size.get().bytes;
 	}
 
-	TraceEvent(SevError, "DDZeroServerLeftInTeam").detail("Team", team->getDesc()).detail("TotalBytesLost", bytesLost);
+	TraceEvent(SevWarnAlways, "DDZeroServerLeftInTeam", self->distributorId)
+	    .detail("Team", team->getDesc())
```

Comment:
add team ID?
` .detail("ServerTeamID", team->getTeamID().shortString())`

Reply:
team id is already part of the string returned by `getDesc()`. Unless you find a dedicated field to be useful, we don't need to log it again.